documenteer/stackdocs/build.py

```diff
@@ -259,6 +266,11 @@ def find_package_docs(package_dir):
                 static_dirs=static_dirs)
 
 
+class NoPackageDocs(Exception):
+    """Exception raised when documentation is not found for an EUPS package.
+    """
```

Comment:
I learned something today.  I though you would have to have a `pass` but declaration of a class only requires an indented block (the docstring in this case).